policy/pa.go

```diff
@@ -174,24 +174,24 @@ func isDNSCharacter(ch byte) bool {
 
 var (
 	errInvalidIdentifier    = berrors.MalformedError("Invalid identifier type")
-	errNonPublic            = berrors.MalformedError("Name does not end in a public suffix")
-	errICANNTLD             = berrors.MalformedError("Name is an ICANN TLD")
-	errPolicyForbidden      = berrors.RejectedIdentifierError("Policy forbids issuing for name")
-	errInvalidDNSCharacter  = berrors.MalformedError("Invalid character in DNS name")
-	errNameTooLong          = berrors.MalformedError("DNS name too long")
-	errIPAddress            = berrors.MalformedError("Issuance for IP addresses not supported")
-	errTooManyLabels        = berrors.MalformedError("DNS name has too many labels")
-	errEmptyName            = berrors.MalformedError("DNS name was empty")
-	errNameEndsInDot        = berrors.MalformedError("DNS name ends in a period")
-	errTooFewLabels         = berrors.MalformedError("DNS name does not have enough labels")
-	errLabelTooShort        = berrors.MalformedError("DNS label is too short")
-	errLabelTooLong         = berrors.MalformedError("DNS label is too long")
-	errMalformedIDN         = berrors.MalformedError("DNS label contains malformed punycode")
-	errInvalidRLDH          = berrors.RejectedIdentifierError("DNS name contains a R-LDH label")
-	errTooManyWildcards     = berrors.MalformedError("DNS name had more than one wildcard")
-	errMalformedWildcard    = berrors.MalformedError("DNS name had a malformed wildcard label")
-	errICANNTLDWildcard     = berrors.MalformedError("DNS name was a wildcard for an ICANN TLD")
-	errWildcardNotSupported = berrors.MalformedError("Wildcard names not supported")
+	errNonPublic            = berrors.MalformedError("Domain name does not end with a valid public suffix (TLD)")
+	errICANNTLD             = berrors.MalformedError("Domain name ends with a reserved ICANN TLD. Certificate can not be issued")
+	errPolicyForbidden      = berrors.RejectedIdentifierError("Policy forbids issuing a certificate for the domain name. Part or all of the name may be blacklisted")
+	errInvalidDNSCharacter  = berrors.MalformedError("Domain name contains invalid character")
+	errNameTooLong          = berrors.MalformedError("Domain name too long. (Note that certificates containing domain names over 230 octets long are not currently supported)")
```

Comment:
I think "octets" is also jargony. How about:

```suggestion
	errNameTooLong          = berrors.MalformedError("Domain name longer than 230 characters")
```

Reply:
I wasn't sure if removing octets might get dev hackles up :)  but 2 issues more seriously.  Is it 230 characters or 230 bytes? (Think non English users and websites, using Unicode to ASCII domain name encoding, for them the limit might be 100 "characters". We shouldn't misinform). And also, enough users will know the limit "ought to be" ~ 253 not 230 who this will.confuse/misinform. The issue is that the see ice can't accept >230, not that >230 is invalid. Maybe this means they need to use another service.....?

Reply:
Yep, good point - I'm happy with using "bytes" consistently here. That said, you've reminded me that we have now done the underlying changes that were needed to fix this limit. I filed an issue to remove the limit: https://github.com/letsencrypt/boulder/issues/4516. For now, let's skip tweaking this message since it will go away soon.

Reply:
Edited, take a look now